src/documentation/text/01_Preface/index.tex

```diff
@@ -0,0 +1,11 @@
+Im Laufe der Zeit schreitet die Technologie im Bereich der Prozessoren von Servern und Computern immer weiter voran.
+Die Prozessoren werden zwar nicht schneller, haben jedoch immer mehr Kerne.
+Um die verschiedenen Kerne der Hardware ausnutzen zu können, wird eine gleichzeitige Codeausführung benötigt.
+Eine Möglichkeit hierzu ist das Aktorenmodell.
+
+Das Aktorenmodell ist grundlegen ein konzeptionelles Modell, das sich mit der gleichzeitigen Ausführung von Codes unter Berücksichtigung von gewissen Regeln beschäftigt.\ifootcite[235]{hewitt1973}
+
+Diese wissenschaftliche Ausarbeitung befasst sich mit den theoretischen Grundlagen des Aktorenmodells und der praktischen Entwicklung dessen im Rahmen der DHBW"=Veranstaltung \enquote{Anwendungsentwicklung 1}.
+Zum Einstieg wird in \autoref{sec:theory} in die Theorie des Aktorenmodells eingeleitet.
+In \autoref{sec:requirements} werden die Anforderungen an die Anwendung genauer ausgeführt, die in \autoref{sec:implementation} in Form der Architektur der Anwendung dargestellt werden.
```

Comment:
`die in \autoref{sec:implementation} in Form der Architektur der Anwendung dargestellt werden.`

Das stimmt so auch nicht. In dem Kapitel wird lediglich beschrieben, wie unsere Anwendung aufgebaut ist.